src/common/service-env.ts

```diff
@@ -0,0 +1,11 @@
+export default class ServiceEnv {
+  key?: string;
```

Comment:
Do we need a key vs. just using the name? Since we have the description for human-readable info, why not just consolidate the name/key fields?

Reply:
I'll remove